into/convert.py

```diff
@@ -79,6 +79,22 @@ def numpy_chunks_to_numpy(c, **kwargs):
     return np.concatenate(list(c))
 
 
+@convert.register(chunks(np.ndarray), np.ndarray, cost=0.5)
+def numpy_to_chunks_numpy(x, chunksize=2**20, **kwargs):
+    return chunks(np.ndarray)(
+            lambda: (x[i:i+chunksize] for i in range(0, x.shape[0], chunksize)))
+
+
+@convert.register(pd.DataFrame, chunks(pd.DataFrame), cost=1.0)
+def chunks_dataframe_to_dataframe(c, **kwargs):
+    return pd.concat(list(c), axis=0)
```

Comment:
In this case I think we only want `axis=0`.  We're not replicating pd.concat generically (one should presumably just use `pd.concat` for that) and logically all of our chunks are row-wise part of the same table.  The other kwargs might be helpful though.